README.md

```diff
@@ -1,42 +1,131 @@
-# sql-data-integrity-checker
+# sdic
```

Comment:
why the acronym? I know you mentioned what it means, but isnt it more obvious and clear having `sql-data-integrity-checker`?